applications/frontend/public/api/models

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Are those files mock data? Should move in a directory named so it's clear it's mock data and moved out from public by using the webpack file loader. The public directory shouldn't be committed when using webpack I think, otherwise it may be unclear what's the source and what's generated

Reply:
yes it's mock data. sure I can move the public assets to within src and use webpack to put it there

Reply:
done